src/main/java/frc/robot/Robot.java

```diff
@@ -24,39 +35,84 @@
   private String m_autoSelected;
   private final SendableChooser<String> m_chooser = new SendableChooser<>();
 
+  // Talons and Victors
   /**
-   * This function is run when the robot is first started up and should be
-   * used for any initialization code.
+   * @TODO add how the ports works
+   */
+  WPI_TalonSRX rightTalon = new WPI_TalonSRX(6);
+  WPI_TalonSRX leftTalon = new WPI_TalonSRX(5);
+  WPI_VictorSPX rightVictor = new WPI_VictorSPX(1);
+  WPI_VictorSPX leftVictor = new WPI_VictorSPX(4);
+
+  // Differential Drive
+  DifferentialDrive differentialDrive = new DifferentialDrive(leftTalon, rightTalon);
+
+  // Joysticks
+  Joystick driveController = new Joystick(0);
+  Joystick manipulateController = new Joystick(1);
+
+  // Drive speed toggles
+  private static final double slowSpeed = 0.5;
+  private static final double highSpeed = 0.75;
+  boolean slowToggle, highToggle = false;
+
+  // Color Sensor
+  private final I2C.Port i2cPort = I2C.Port.kOnboard;
+  private final ColorSensorV3 m_colorSensor = new ColorSensorV3(i2cPort);
+  private final ColorMatch m_colorMatcher = new ColorMatch();
+  private final Color kBlueTarget = ColorMatch.makeColor(0.143, 0.427, 0.429);
+  private final Color kGreenTarget = ColorMatch.makeColor(0.197, 0.561, 0.240);
+  private final Color kRedTarget = ColorMatch.makeColor(0.561, 0.232, 0.114);
+  private final Color kYellowTarget = ColorMatch.makeColor(0.361, 0.524, 0.113);
+  boolean colorSensorDetectGreen, colorSensorDetectRed, colorSensorDetectBlue, colorSensorDetectYellow = false;
+  boolean colorWheelMode = false;
```

Comment:
By convention, booleans are often named using "is" so that their purpose is clearer, and that `if` statements read more easily. For example, `colorSensorDetectGreen` -> `isLookingForGreen`, `colorWheelMode ` -> `isInControlPanelMode` (the control panel is the accurate name for the color wheel). For more info on this convention, see [here](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig).

Reply:
Implemented.